_posts/convars/2021-9-24-sv_airdecelerate.md

```diff
@@ -0,0 +1,13 @@
+---
+title: sv_airdecelerate
+category: var
+tags:
+  - conc
+  - movement
+  - player
+requires_mapping: true
+---
+
+Like `sv_airaccelerate`, but for air slowdown only (`+back`).
+
+Default command value is the same as the mode's default `sv_airaccelerate` value in all gamemodes except for Conc, where it is set to `10`
```

Comment:
Bit of a nitpick but this is a variable not a command
```suggestion
Default value is the same as the mode's default `sv_airaccelerate` value in all gamemodes except for Conc, where it is set to `10`
```

Reply:
:+1: